Extensions/xkit_preferences.css

```diff
@@ -5,7 +5,24 @@
 	--xkit-on-primary: black;
 	--xkit-secondary: rgb(240,240,240);
 	--xkit-on-secondary: rgb(120,120,120);
-	--xkit-on-color: white;
+	--xkit-on-pale: black;
+	--xkit-on-saturated: white;
+
+	--xkit-05-overlay: rgba(0,0,0,.05);
+	--xkit-11-overlay: rgba(0,0,0,.11);
+	--xkit-22-overlay: rgba(0,0,0,.22);
+	--xkit-33-overlay: rgba(0,0,0,.33);
+	--xkit-44-overlay: rgba(0,0,0,.44);
+	--xkit-53-overlay: rgba(0,0,0,.53);
+	--xkit-83-overlay: rgba(0,0,0,.83);
+
+	--xkit-white-05-overlay: rgba(255,255,255,.05);
+	--xkit-white-11-overlay: rgba(255,255,255,.11);
+	--xkit-white-22-overlay: rgba(255,255,255,.22);
+	--xkit-white-33-overlay: rgba(255,255,255,.33);
+	--xkit-white-44-overlay: rgba(255,255,255,.44);
+	--xkit-white-53-overlay: rgba(255,255,255,.53);
+	--xkit-white-83-overlay: rgba(255,255,255,.83);
```

Comment:
instead of specifying all of these colors individually, i think it makes more sense to specify a single "xkit-overlay-base" css variable, and then use it in situ as follows:

```css
--xkit-overlay-base: 255, 255, 255;

.....

.foo {
  background: rgba(var(--xkit-overlay-base), .53);
}
```

(for more examples, see https://stackoverflow.com/questions/40010597/how-do-i-apply-opacity-to-a-css-color-variable/41265350#41265350)

Reply:
That's smart!! I'll implement it now.